WordPress/src/main/res/layout/edit_post_header.xml

```diff
@@ -0,0 +1,31 @@
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="wrap_content"
+    android:layout_height="match_parent"
+    android:orientation="horizontal"
+    android:gravity="center_vertical"
+    android:contentDescription="@string/menu_close_editor"
+    android:foreground="?attr/selectableItemBackground"
+    android:clickable="true"
+    android:focusable="true">
+
+    <ImageView
+        android:layout_width="@dimen/edit_post_header_chevron_size"
+        android:layout_height="@dimen/edit_post_header_chevron_size"
+        android:importantForAccessibility="no"
+        android:src="@drawable/chevron_down"
```

Comment:
❓ I didn't quite understand why we are using a `chevron_down` icon here as the back button, shouldn't it at least be a `chevron_left` or the regular Android back icon (arrow)? 

As a user, this new icon didn't make me think that button would close the editor, but rather present me some options or something like that.

Reply:
Good question, I think it makes more sense on iOS since the View gets dismissed by being animated from the top to the bottom. I'll check with the designers! Thanks!

Reply:
Hey @thomashorta 👋  after checking with the designers we are going with the `chevron_left` so I've added it and removed the other one in https://github.com/wordpress-mobile/WordPress-Android/pull/18705/commits/9c1e6afdacb7fbf89ae3c87e730f3530b17b8952

Let me know if you have any other questions, thanks!